tests/mpd/protocol/test_playback.py

```diff
@@ -80,37 +80,37 @@ def test_repeat_on_without_quotes(self):
 
     def test_setvol_below_min(self):
```

Comment:
IIRC, the file split in `.../mpd/protocol` is by sections of the MPD protocol docs, not parts of our core API.